### ISSUE TITLE
npm install is failing with 'invalid syntax' error on Windows

### ISSUE THREAD

**Issue by emirhosseini** [kafka-lens/kafka-lens#56]
npm install is failing with the following error:

```
gyp ERR! stack Error: Command failed: C:\Python38\python.EXE -c import sys; print "%s.%s.%s" % sys.version_info[:3];
gyp ERR! stack   File "<string>", line 1
gyp ERR! stack     import sys; print "%s.%s.%s" % sys.version_info[:3];
gyp ERR! stack                       ^
gyp ERR! stack SyntaxError: invalid syntax
gyp ERR! stack
gyp ERR! stack     at ChildProcess.exithandler (child_process.js:289:12)
gyp ERR! stack     at ChildProcess.emit (events.js:182:13)
gyp ERR! stack     at maybeClose (internal/child_process.js:962:16)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:251:5)
gyp ERR! System Windows_NT 10.0.14393
gyp ERR! command "C:\\Program Files\\nodejs\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
gyp ERR! cwd C:\tools\kafka-lens\node_modules\node-rdkafka
```

**Comment by iAmHarmon** [comment]
Hi there,

We have not tested a Windows build of this project. I can definitely give this a shot when I get home to see what the issue is. Will update at that point.

Cheers!

**Comment by nahuelarjonadev** [comment]
This has been reviewed and fixed in version **2.0.0**. Should normally work now on Windows, Linux, and macOS
